### DISCUSSION TITLE
Who wrote the play "Hamlet" 📑?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#70] [Galaxy Qs] [answered]
Who wrote the play "Hamlet"?

**Comment by Denisyasyuchenya** [comment, marked as answer]
William Shakespeare

**Comment by mitchiemt11** [comment]
William Shakespeare. It is one of his most famous and frequently performed works, believed to have been written between 1599 and 1601.

@Sriansh-raj Though Shakespeare himself is no longer living, his writing style still captivates me. I hold a strong liking for his plays, with "Julius Caesar" being a personal favourite. The way he constructed his words is truly remarkable.
